cloudinary_cli/core/admin.py

```diff
@@ -19,11 +19,16 @@
 @option("-o", "--optional_parameter", multiple=True, nargs=2, help="Pass optional parameters as raw strings.")
 @option("-O", "--optional_parameter_parsed", multiple=True, nargs=2,
         help="Pass optional parameters as interpreted strings.")
+@option("--recursive", nargs=1, help="Query a field in the response recursively.", default=None)
+@option("--cursor", nargs=1, help="Cursor field to use when using the --recursive flag.", default=None)
```

Comment:
What are the options here? I believe it should be set to `next_cursor` by default. And a more clear name would be `cursor-field`, someone might expect to put the actual `next_cursor` value here.